tests/include.rs

```diff
@@ -110,6 +317,33 @@ fn should_include_u64() {
     assert!(included == expected, "included doesn't match expected file output");
 }
 
+#[test]
+fn should_include_u64le() {
+    let included = include_bytes!("tests/include.in" as u64le);
+    let expected = read("tests/include.in").expect("To read source code");
+    let expected = unsafe {
+        slice::from_raw_parts(expected.as_ptr() as *const u64, expected.len() / 8)
+    };
+    assert!(included == expected, "included doesn't match expected file output");
+}
+
+#[test]
+fn should_include_u64be() {
+    let included = include_bytes!("tests/include.in" as u64be);
+    let mut expected = read("tests/include.in").expect("To read source code");
+    // swap bytes
+    for i in 0..expected.len() / 8 {
```

Comment:
Personally I think it would be more cleaner to just reinterpret `expected` and then iterate over every integer swapping bytes

Swapping raw bytes as it is doesn't look all that good

Reply:
I agree that this is not very clean. Can you tell me what code you have in mind for reinterpreting `expected`?

Reply:
Just simply as:

```rust
let mut expected = read("tests/include.in").expect("To read source code");
let expected = unsafe {  slice::from_raw_parts(expected.as_ptr() as *const u16, expected.len() / 2) };
for integer in expected.iter_mut() {
    *integer = integer.swap();
}
```

Reply:
Oh, cool! I did not know about `swap_bytes`.

Reply:
Oh yeah, it is `swap_bytes` not `swap` 😄